pkg/controllers/cspc-controller/sync.go

```diff
@@ -115,9 +115,9 @@ func (c *Controller) sync(cspc *cstor.CStorPoolCluster, cspiList *cstor.CStorPoo
 
 	if ok, reason := c.ShouldReconcile(*cspc); !ok {
 		// Do not reconcile this cspc
-		message := fmt.Sprintf("Cannot not reconcile CSPC %s as %s", cspc.Name, reason)
+		message := fmt.Sprintf("Can not reconcile CSPC %s as %s", cspc.Name, reason)
```

Comment:
nit: Since reason is shown as events under a particular CSPC, So we no need to even log the CSPC name